src/components/Scene.js

```diff
@@ -8,18 +8,63 @@ import { Ground } from "./Ground";
 import { OtherPlayers } from "./OtherPlayers";
 import settings from "../devOnline";
 import { FormCubeArrays } from "./cubecomps/FormCubeArrays";
+import { Cubes } from "./cubecomps/Cubes";
+import { useRef } from "react";
 
 
 extend({ RenderPixelatedPass });
 
 export const Scene = ({ activeTextureREF }) => {
+  console.log('----this is scene')
   const { size, scene, camera } = useThree();
   const resolution = useMemo(() => new THREE.Vector2(size.width, size.height), [size]);
+  const REF_ALLCUBES = useRef({"0.0.0": { pos: [0, 0, 0], texture: "log" }})
+
+  const shiftpos=useRef([50,10,10])
+  const currblockpos=useRef([0,0])
+  let shiftspeed=.01
+  const lookatatinit = useRef(true)
+
+
+
+
+  /*
+    adjusting camera: to help see a larger amount of cubes from different views
+  */
+  useFrame((state)=>{
+  //   console.log(
+  //     Math.floor(camera.position.x/16)
+  //     ,
+  //     Math.floor(camera.position.z/16)
+  //     )
+
+  //   //larger birds eye
```

Comment:
maybe lets add larger and slant as options we pass in props or maybe by something in game?